prow/cluster/private/authentikos_rbac.yaml

```diff
@@ -0,0 +1,44 @@
+# Copyright 2019 Istio Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+kind: ServiceAccount
```

Comment:
nit: I think it is technically better to include the ServiceAccount in the `_deployment.yaml` file before the Deployment so that the Deployment can be applied on its own or before applying the `_rbac.yaml` file.

Reply:
Makes sense to me. I made the update!